src/main/java/seedu/address/model/LocalCourseCatalogue.java

```diff
@@ -0,0 +1,132 @@
+package seedu.address.model;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import javafx.collections.ObservableList;
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.model.localcourse.LocalCourse;
+import seedu.address.model.localcourse.UniqueLocalCourseList;
+
+/**
+ * Wraps local course data at the catalogue level
+ * Duplicates are not allowed (by .isSameLocalCourse comparison)
+ */
+@SuppressWarnings("checkstyle:CommentsIndentation")
```

Comment:
TBD: Let's try our best not to suppresswarnings

Reply:
Thanks for pointing this out, must have forgotten to remove it.